### DISCUSSION TITLE
Feature request: Support SGTL5000 and Teensy Audio Shield

### DISCUSSION THREAD

**Discussion by madskjeldgaard** [pschatzmann/arduino-audio-driver#31] [Ideas]
Hi!

thanks for making this fabulous library!

Just an idea:

The Teensy Audio Shield is quite popular, and it would be a great addition to this fantastic library to support it as well at some point. 

The board has an SD card reader and a SGTL5000 codec.

You can read about the board here:
https://www.pjrc.com/store/teensy3_audio.html

The driver has already been written with the Teensy eco system in mind here, so it shouldn't be too difficult to port it.

https://github.com/PaulStoffregen/Audio/blob/master/control_sgtl5000.cpp

**Comment by pschatzmann** [comment]
I was considering it but I found it a bit too expensive.
Feel free to extend my [audio-driver](https://github.com/pschatzmann/arduino-audio-driver) library

**Comment by madskjeldgaard** [comment]
I might do that at some point, thanks :)

**Comment by SmittyHalibut** [comment]
Found this before filing my issue just now.  Heads up @madskjeldgaard looks like SGTL5000 was added in the 0.3.0 release just a few days ago.  However, I'm not able to get it working yet.  See #73
